src/main/java/core/basesyntax/db/StorageImpl.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageImpl {
```

Comment:
Consider making the StorageImpl class an interface or an abstract class, as it would provide better flexibility for future implementations.